app/controllers/carts_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class CartsController < ApplicationController
+  before_action :initialize_session
+  decorates_assigned :cart_items
+
+  def index
+    @cart_items = Menu.where(id: session[:cart])
+  end
+
+  def create
+    AddToCartService.call(menu_id: params['cart']['id'].to_i,
```

Comment:
you could pass `params` and `session` into the service to avoid these manipulations in controller
also,  `['cart']['id']` => `dig(:card, :id)`